### ISSUE TITLE
Snippet improve for loop

### ISSUE THREAD

**Issue by programmer04** [microsoft/vscode-go#2943]
Now snippet for `for` looks like that:
```
for index := 0; index < count; index++ {
		
}
```
but in most languages and in Go too, using `i` instead of `index` is a convention:
```
for i := 0; i < count; i++ {
		
}
```
for example, this is recommended here https://talks.golang.org/2014/names.slide#6
What do you think about it? I'll be happy to make a Pull Request 🙂

**Comment by ramya-rao-a** [comment]
Thanks for reporting @programmer04 

@stamblerre, @hyangah Thoughts?

**Comment by stamblerre** [comment]
That sounds quite reasonable to me. Short variable names are recommended in Go.

**Comment by ramya-rao-a** [comment]
@programmer04 Please do go ahead and make a Pull Request :)

**Comment by programmer04** [comment]
Thanks for answer, I made the PR #3010 🙂

**Comment by ramya-rao-a** [comment]
This issue is fixed in the latest version (0.13.0) of this extension

Thanks for reporting and for the PR @programmer04 !

Happy Coding!
